[MINOR][DOCS] add link for Debugging your Application in running-on-yarn.html#launching-spark-on-yarn

brandonJY opened the pull request:
### What changes were proposed in this pull request?
 add link for Debugging your Application in `running-on-yarn.html#launching-spark-on-yar`



### Why are the changes needed?
Currrently on running-on-yarn.html page launching-spark-on-yarn section, it mentions to refer for Debugging your Application. It is better to add a direct link for it to save reader time to find the section
  ![image](https://user-images.githubusercontent.com/20021316/87867542-80cc5500-c9c0-11ea-8560-5ddcb5a308bc.png)





### Does this PR introduce _any_ user-facing change?
Yes. 
Docs changes.  
1. add link for Debugging your Application in `running-on-yarn.html#launching-spark-on-yarn` section
Updated behavior:
![image](https://user-images.githubusercontent.com/20021316/87867534-6eeab200-c9c0-11ea-94ee-d3fa58157156.png)
2. update Spark Properties link to anchor link only



### How was this patch tested?
manual test has been performed to test the updated

brandonJY committed:
add link for Debugging your Application

add link for Debugging your Application as the text is mentioned in Launching Spark on YARN

AmplabJenkins commented:
Can one of the admins verify this patch?

AmplabJenkins commented:
Can one of the admins verify this patch?

srowen commented on docs/running-on-yarn.md:
```diff
@@ -60,7 +60,7 @@ For example:
         10
 
-The above starts a YARN client program which starts the default Application Master. Then SparkPi will be run as a child thread of Application Master. The client will periodically poll the Application Master for status updates and display them in the console. The client will exit once your application has finished running.  Refer to the "Debugging your Application" section below for how to see dri ...
+The above starts a YARN client program which starts the default Application Master. Then SparkPi will be run as a child thread of Application Master. The client will periodically poll the Application Master for status updates and display them in the console. The client will exit once your application has finished running.  Refer to the [Debugging your Application](running-on-yarn.html#debugging-y ...
```
This is fine, but does the anchor link alone work? `(#debugging-your-application)`

brandonJY replied:
@srowen Thanks for the suggestion.  Tested, the anchor link alone works. Besides this link, updated another link on this page which is simliar as well.

srowen commented:
Jenkins test this please

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder-K8s/30745/
Test PASSed.

SparkQA commented:
**[Test build #126133 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/126133/testReport)** for PR 29154 at commit [`f0f5793`](https://github.com/apache/spark/commit/f0f579389c3a7f0662915c9cbf08228786061e68).

SparkQA commented:
**[Test build #126133 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/126133/testReport)** for PR 29154 at commit [`f0f5793`](https://github.com/apache/spark/commit/f0f579389c3a7f0662915c9cbf08228786061e68).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/126133/
Test PASSed.

brandonJY committed:
Update anchor link alone for links on running-on-yarn pages

[DOCS] For links that refer on the same page in running-on-yarn pages, update to anchor link alone

brandonJY commented:
Jenkins test this please

srowen reviewed (approved):
Jenkins test this please

HyukjinKwon commented:
Jenkins test this please

SparkQA commented:
**[Test build #126217 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/126217/testReport)** for PR 29154 at commit [`96fa931`](https://github.com/apache/spark/commit/96fa9318bfb2a4d94f0f2347f4f2b86e81940143).

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder-K8s/30831/
Test PASSed.

SparkQA commented:
**[Test build #126217 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/126217/testReport)** for PR 29154 at commit [`96fa931`](https://github.com/apache/spark/commit/96fa9318bfb2a4d94f0f2347f4f2b86e81940143).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/126217/
Test PASSed.

HyukjinKwon commented:
Merged to master, branch-3.0 and branch-2.4.
